modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -1293,4 +1293,41 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  test("-D.. options passed to the child app") {
+    val inputs = TestInputs(
+      Seq(
+        os.rel / "Hello.scala" -> """
+                                    |import java.lang.System
```

Comment:
No need to import this, `java.lang.*` is automatically imported in Scala sources.